compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -202,13 +207,15 @@ object Inliner {
       case info: PolyType => info.instantiate(targs.map(_.tpe))
       case info => info
 
-    val unappplySym = newSymbol(cls, sym.name.toTermName, Synthetic | Method, unapplyInfo, coord = sym.coord).entered
-    val unapply = DefDef(unappplySym, argss =>
-      inlineCall(fun.appliedToArgss(argss).withSpan(unapp.span))(using ctx.withOwner(unappplySym))
+    val unapplySym = newSymbol(cls, sym.name.toTermName, Synthetic | Method, unapplyInfo, coord = sym.coord).entered
```

Comment:
Where did the inlineCall go? Don't we need that?

Reply:
We don't need to expand it at this point, the inlining phase takes care of inlining the code.